terraform/modules/happy-service-eks/main.tf

```diff
@@ -458,7 +474,14 @@ resource "kubernetes_deployment_v1" "deployment" {
                 read_only  = true
               }
             }
-
+            dynamic "volume_mount" {
+              for_each = toset(var.emptydir_volumes)
+              content {
+                # TODO FIXME do we want the mount path to be configurable????
```

Comment:
@rzlim08 do you think we should add support for specifying mount paths, or delete these comments?

Reply:
IMO there's no immediate need for it and a slight potential for the footgun of mounting over necessary folders, so I'll delete the comments. It'd be relatively easy to change in the future though.